### ISSUE TITLE
Canceled debug request error is not localized

### ISSUE THREAD

**Issue by Zebsterpasha** [eclipse-theia/theia#12709]
### Steps to Reproduce:

https://github.com/eclipse-theia/theia/blob/e52fc5b296d5d5d0f586d3fffe94e50fa795afc8/packages/debug/src/browser/debug-session-connection.ts#L156

https://github.com/eclipse-theia/theia/blob/e52fc5b296d5d5d0f586d3fffe94e50fa795afc8/packages/debug/src/browser/debug-session-connection.ts#L200

https://github.com/eclipse-theia/theia/blob/e52fc5b296d5d5d0f586d3fffe94e50fa795afc8/packages/debug/src/browser/debug-session-connection.ts#L208

### Additional Information

- Operating System: Any
- Theia Version: The latest

**Comment by vince-fugnitto** [comment]
@Zebsterpasha I don't believe we (or vscode for that matter) localize any thrown errors, only user-facing strings.

**Comment by Zebsterpasha** [comment]
> @Zebsterpasha I don't believe we (or vscode for that matter) localize any thrown errors, only user-facing strings.

Yeah, I know, but I received this message in UI, here what I got:

![image](https://github.com/eclipse-theia/theia/assets/77332790/65065180-3e6d-417b-8a57-e82142f120e2)

I got this error when thied to run the debugger, but it was not available to run (there is an application based on theia)
